tests/V0SignedAUthenticationUrlTest.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+use Creads\Partners\V0SignedAuthenticationUrl;
+use PHPUnit\Framework\TestCase;
+
+final class V0SignedAuthenticationUrlTest extends TestCase
+{
+    //Client APP_ID: 15_64d081gwqckkwwkc0s0occ4ckss8ww04o4ow80k0444kkkg4g0
+    //Client SECRET_KEY: 27ldyvaorh34coc00kggkogscgwc8s84go8ck8wks4ogsgg8kk
+    //bin/partners signed-auth-url  "My Organization" j.doe@creads.org --protocol=0
```

Comment:
Actually, the first parameters is the user email, not the organization name. Like this, the URL could be well generated and validated but sounds not well formatted.